docs/conformance-tests-lib.md

```diff
@@ -0,0 +1,110 @@
+# Conformance Tests
+
+`interchaintest` comes with a suite of conformance tests. These tests ensure IBC and relayer compatibility. On a high-level it tests:
+- `client`, `channel`, and `connection` creation
+- messages are properly relayed and acknowledged
+- packets are being properly timed out
+
+You can view all the specific conformance test by reviewing them in the [conformance](../conformance/) folder.
+
+## Importing Conformance Tests In Your Project
+
+`interchaintest` can be imported into your own packages to be used as a library as well as being used from the 
+binary itself, see [here](conformance-tests-bin.md).
+
+`interchaintest` does not currently follow semver due to the tightly coupled nature of the package with ibc-go,
+as a result there are no tagged versions available for importing. To import `interchaintest` into your own repo
```

Comment:
this is about to not be the case. Once the Proposal v50 PR is fixed and in, v7.0.0 and v8.0.0 will be tagged for local-interchain

Reply:
ty, i'll just kill this entire part here then!